README.md

```diff
@@ -1,21 +1,34 @@
 # cloudflare-worker-template
-Bootstrap your Cloudflare Worker to connect to a pgEdge database
+Bootstrap your Cloudflare Worker to connect to a pgEdge database.
 
 [![Deploy to Cloudflare Workers](https://deploy.workers.cloudflare.com/button)](https://deploy.workers.cloudflare.com/?url=https://github.com/pgEdge/cloudflare-worker-template)
 
 This is a simple Cloudflare Worker that connects to a pgEdge database. Use it as a starting point 
-for your own project. 
+for your own project! Simply click on the button above to deploy this worker to your Cloudflare
+account and create a Github repository for it.
 
 ## pgEdge connection
 
-To find your pgEdge connection string, go to the [pgEdge console](https://app.pgedge.com). If you have 
-not yet signed up for pgEdge, you can easily sign in with your social accounts. Creating a new database 
-is free and takes less than a minute.
+The easiest way to get your worker connected to a pgEdge database is to go to the [pgEdge console](https://app.pgedge.com).
+If you have not yet signed up for pgEdge, you can easily sign in with your social accounts. Creating a new 
+database is free and takes less than a minute.
 
-Once you've reached the console, navigate to the database you want to connect to. In the `Connect to your database` 
-section, you will find your username, password, and the host is the `Domain`.  
+Once you've connected the console, navigate to the database you want to connect to. On the sidebar, click
+on the "Integrations" tab. You should see a "Cloudflare Workers" integration. This will provide you with
+all the details you need to connect your worker to your database.
 
-## DB connection
+If you prefer to follow READMEs instead of clicking around, here are the steps to get your worker connected...
+
+### Setup your local environment
+
+Once you've deployed your worker and cloned your repository, you'll need to setup your local environment to 
+be able to run the worker locally. You'll need to install wrangler, the Cloudflare CLI tool.
+
+```bash
+npm install -g @cloudflare/wrangler
```

Comment:
Is it worth adding this to the dev dependencies? Then they can just do `npm i` and run wrangler with `npx`.

It would also be helpful to mention that you need node >= v16.13.0.

Reply:
Oddly this is not generally a pattern for these kinds of repos.  If you go to Cloudflare's [Deploy to Workers](https://deploy.workers.cloudflare.com/) sample page, the [repos](https://github.com/cloudflare/workers-graphql-server) [there](https://github.com/adamschwartz/web.scraper.workers.dev) [don't](https://github.com/eidam/cf-workers-status-page) [have](https://github.com/GregBrimble/cf-workers-typescript-template) it as a dependency for the most part.

We don't need to follow that lead, but it does seem to be a staple of wrangler-driven development.

Reply:
Their installation instructions say to install it as a dev dependency FWIW: https://developers.cloudflare.com/workers/wrangler/install-and-update/